Development dependencies.md

```diff
@@ -6,3 +6,11 @@ Building this project requires:
   * Assumes the Inno Setup dir, by default "%ProgramFiles(x86)%\Inno Setup 5" is in your path.
 * 7Zip from http://www.7-zip.org/download.html
   * Assumes the 7Zip dir, by default "C:\Program Files\7-Zip" is in your path.
+
+Locally running EDDI:
+
+* Start Visual Studio and choose "Open a project/solution" then select the "EDDI.sln" solution file
+* In the top toolbar change "Debug" to "Release"
+* In the Solution Explorer, right-click "Eddi" and choose "Set as Startup Project"
+* In the "Build" menu choose "Build Solution" (or use the keyboard shortcut Ctrl-Shift-B)
+* In the top toolbar press the "Start" button
```

Comment:
Is the intent here to describe how to generate the install files or how to run EDDI from Visual Studio?

If the intent is to describe how to run EDDI locally from within Visual Studio, I'd recommend using a "Debug" build rather than a "Release" build. 

If the intent is to describe how to generate the install files then I think we may to clarify the intent and the output location for the files?

Reply:
The intent was purely how to describe to run EDDI locally built from sources (as referred to in the readme). I'll remove the line about switching from "Debug" to "Release".